### PR TITLE
Removing hard-coded public keys from Manufacturer.

### PR DESCRIPTION
**Pull request by DukeDavis12**
Changes Include:
        - Removing hard-coded owner & reseller public key from the manufacturer.
        - logic for loading owner & reseller public during runtime.
        - new sample .pem files of owner and reseller.

### CODE REVIEW

File: component-samples/demo/manufacturer/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ WORKDIR /home/manufacturer/
 COPY ./lib ./lib/
 COPY ./certs ./certs/
 COPY manufacturer.jar .
-COPY manufacturer_keystore.p12 .
+COPY manufacturer_keystore.p12 owner.pem* reseller.pem* .
```
Comment (trbehera) [inline component-samples/demo/manufacturer/Dockerfile:20]: This line should be changed as below.

COPY manufacturer_keystore.p12 owner_pub_keys.pem reseller_pub_keys.pem ./

File: component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java [hunk 2]
```diff
@@ -227,9 +193,21 @@ protected void failed(Exception e) {
     DiDbStorage db = new DiDbStorage(cs, ds, keyResolver, ods);
     DiDbManager manager = new DiDbManager();
     manager.createTables(ds);
-    manager.addCustomer(ds, 1, "owner", ownerKeysPem);
-    manager.addCustomer(ds, 2, "reseller", resellerKeysPem);
-    manager.setAutoEnroll(ds, 1);
+    try {
```
Comment (trbehera) [inline component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java:196]: You can combine both the key additions in the same try block.

Also, the log message for successful/failed addition should be as below.

"Registered public keys for customer 'owner'"
"Registered public keys for customer 'reseller'"

"No default public keys found for customer 'owner'"
"No default public keys found for customer 'reseller'"

**Review by trbehera — CHANGES_REQUESTED** [review]

File: component-samples/demo/manufacturer/Dockerfile (hunk 1 above)
Comment (DukeDavis12) [reply component-samples/demo/manufacturer/Dockerfile:20]: Done.

File: component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java (hunk 2 above)
Comment (DukeDavis12) [reply component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java:196]: Done

**Review by trbehera — DISMISSED** [review]

**Review by trbehera — APPROVED** [review]

File: component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java (hunk 2 above)
Comment (DukeDavis12) [reply component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java:198]: Two separate try blocks are used here as Customer `owner` and 'reseller` are independent cases. There can be cases where the owner key is not present and the reseller keys are present, So we if combine them in the same try block, we might miss updating reseller keys due to the exception raised while loading the owner keys.

**Review by grbean — APPROVED** [review]

**Pull request merged**